runtime/plugins/console/bootstrap.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# Fail on error
+set -e
+
+export PHP_INI_SCAN_DIR="/opt/php-ini/:/var/task/php/conf.d/"
```

Comment:
On master we didn't need a `bootstrap.sh` file (the `bootstrap` file was a PHP file directly) -> do we need it now to set the `PHP_INI_SCAN_DIR` variable?

I'm guessing we are now forced to set `PHP_INI_SCAN_DIR` manually because we don't compile PHP anymore, is that the reason? Is there any other way we can set `PHP_INI_SCAN_DIR` once? (I don't think there is, but just making sure)

Reply:
I replied to this comment here: https://github.com/brefphp/bref/tree/runtime/runtime#bootstrapsh-file and I also added some comments to the file itself, let me know what you think